criu-3.15/lib/py/cli.py

```diff
@@ -30,10 +35,7 @@ def outf(opts):
 
 
 def dinf(opts, name, mode=None):
-    if not mode:
-        return open(os.path.join(opts['dir'], name))
-    else:
-        return open(os.path.join(opts['dir'], name), mode)
+    return open(os.path.join(opts['dir'], name), mode='rb')
```

Comment:
Please don't change function definitions? Maybe the correct commits were not pulled? Or merge conflicts were not correctly resolved.

Reply:
Possibly some issue with an older merge. The code, however, works for both crit recode and ss. I can take the code snippet from the mainline to avoid any conflicts.

Reply:
Yes, keep mainline code wherever poss